requirements-common.txt

```diff
@@ -22,4 +22,6 @@ outlines >= 0.0.43, < 0.1 # Requires torch >= 2.1.0
 typing_extensions >= 4.10
 filelock >= 3.10.4 # filelock starts to support `mode` argument from 3.10.4
 pyzmq
+librosa # Required for audio processing
+soundfile # Required for audio processing
```

Comment:
Can you mock these in `vllm/docs/source/conf.py`? I think otherwise it'll break our doc. Thanks!

Reply:
No problem -- sorry to miss that!